app/src/main/java/com/ivy/wallet/utils/UtilExt.kt

```diff
@@ -3,6 +3,7 @@ package com.ivy.wallet.utils
 import android.app.KeyguardManager
 import android.content.Context
 import android.icu.util.Currency
+import com.google.firebase.crashlytics.internal.model.ImmutableList
```

Comment:
⚠️ That's some Firebase Crashlytics `ImmutableList` - we need the immutable list from this KotlinX library https://github.com/Kotlin/kotlinx.collections.immutable! Otherwise, the Compose optimization won't work.